comtypes/server/register.py

```diff
@@ -37,18 +37,19 @@
 """
 
 import _ctypes
-import ctypes
 import logging
-import os
 import sys
 import winreg
-from ctypes import WinError, windll
+from ctypes import WinDLL, WinError
+from ctypes.wintypes import HKEY, LONG, LPCWSTR
+from os.path import abspath, basename, dirname, splitext
```

Comment:
I think it's fine to leave `os.path.foobar` as is. For #735, let's limit it to changes in the import and reference methods from `ctypes` and `comtypes`, and move the rest to a separate scope.